pkg/auth/client.go

```diff
@@ -2,10 +2,16 @@ package auth
 
 import (
 	"context"
+	"errors"
 
 	"github.com/containerd/containerd/remotes"
 )
 
+// Common errors
+var (
+	ErrNotLoggedIn = errors.New("not_logged_in")

```

Comment:
since this is a human-facing error code, would it be better to make this more human readable? "not logged in" might be a better error message than "not_logged_in", especially considering clients like Helm will be returning this error message as output.